### PR TITLE
courier-prime: init at unstable-2019-12-05

### PR DESCRIPTION
**Pull request by austinbutler**
###### Motivation for this change

I'm intending to add a tool called [`wrap`](https://github.com/Wraparound/wrap) that requires this font (or a similar one, but Courier Prime is considered the top font for screenplays). Aside from needing it as an input to that package, I figure it's useful to have available as a standalone font as well.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 119190` at dc7afbf7 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>courier-prime</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`installPhase` should probably contain `runHook preInstall` and `runHook postInstall`.

Near pkgs/data/fonts/courier-prime/default.nix:14:3:
```
   |
14 |   installPhase = ''
   |   ^
```
</li>
  </ul>
</details>

File: pkgs/data/fonts/courier-prime/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "courier-prime";
+  version = "2019-11-20";
```
Comment (SuperSandro2000) [inline pkgs/data/fonts/courier-prime/default.nix:5]:
```suggestion
  version = "unstable-2019-11-20";
```

File: pkgs/data/fonts/courier-prime/default.nix [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
```
Comment (SuperSandro2000) [inline pkgs/data/fonts/courier-prime/default.nix:3]: Can we use only fetchFromGitHub here?

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/data/fonts/courier-prime/default.nix (hunk 2 above)
Comment (austinbutler) [reply pkgs/data/fonts/courier-prime/default.nix:3]: I don't understand what you mean.

Comment (SuperSandro2000) [reply pkgs/data/fonts/courier-prime/default.nix:3]: Like https://github.com/NixOS/nixpkgs/blob/master/pkgs/data/fonts/amiri/default.nix does it.

Comment (austinbutler) [reply pkgs/data/fonts/courier-prime/default.nix:3]: Ah I see, you mean _not_ using `fetchFromGitHub`. What benefit does that provide? Would the project need to have any releases or tags for that to work? This project [doesn't have either](https://github.com/quoteunquoteapps/CourierPrime/releases).

Comment (SuperSandro2000) [reply pkgs/data/fonts/courier-prime/default.nix:3]: > What benefit does that provide?

It is less complex and if you are just copying files you don't need a full blown stdenv.

> Would the project need to have any releases or tags for that to work?

You are not using one right now, so no.

**Comment by austinbutler** [comment]
Result of `nixpkgs-review pr 119190` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>courier-prime</li>
  </ul>
</details>

File: pkgs/data/fonts/courier-prime/default.nix (hunk 2 above)
Comment (austinbutler) [reply pkgs/data/fonts/courier-prime/default.nix]: Changed.

File: pkgs/data/fonts/courier-prime/default.nix [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{ lib, fetchzip }:
+
+let
+  version = "unstable-2019-12-05";
+  repo = "CourierPrime";
+  rev = "7f6d46a766acd9391d899090de467c53fd9c9cb0";
+in fetchzip rec {
+  name = "courier-prime";
+  url =
+    "https://github.com/quoteunquoteapps/${repo}/archive/${rev}/${name}.zip";
```
Comment (SuperSandro2000) [inline pkgs/data/fonts/courier-prime/default.nix:10]:
```suggestion
  url = "https://github.com/quoteunquoteapps/${repo}/archive/${rev}/${name}.zip";
```

File: pkgs/data/fonts/courier-prime/default.nix [hunk 4]
```diff
@@ -0,0 +1,24 @@
+{ lib, fetchzip }:
+
+let
+  version = "unstable-2019-12-05";
+  repo = "CourierPrime";
+  rev = "7f6d46a766acd9391d899090de467c53fd9c9cb0";
+in fetchzip rec {
+  name = "courier-prime";
```
Comment (SuperSandro2000) [inline pkgs/data/fonts/courier-prime/default.nix:8]:
```suggestion
  name = "courier-prime-${version}";
```

**Pull request merged**